### PR TITLE
backend/namma-dsl: fixed warnings

### PR DESCRIPTION
**Pull request by roman-bodavskiy**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [x] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Backend/nix/scripts.nix [hunk 1]
```diff
@@ -44,7 +44,20 @@ _:
         exec = ''
           set -x
           cd ./Backend  # These processes expect $PWD to be backend, for reading dhall configs
-          cabal run alchemist-generator-exe -- "$@"
```
Comment (roman-bodavskiy) [inline Backend/nix/scripts.nix:47]: @ErasedSoul I've tested this command on my Linux machine and it did not worked. Did it work for you? 
I suggest to add new command `, run-generator-all`. Please check my changes

File: Backend/app/alchemist/src/Main.hs [hunk 2]
```diff
@@ -67,7 +67,7 @@ getFilePathsForConfiguredApps rootDir = do
 main :: IO ()
 main = do
   putStrLn ("Version " ++ NammaDSL.version)
-  generateAllSpecs <- ("--all" `elem`) <$> getArgs
+  generateAllSpecs <- ("all" `elem`) <$> getArgs
```
Comment (ErasedSoul) [inline Backend/app/alchemist/src/Main.hs:70]: --all was working for me ..
I was doing 
`, run-generator --all`

Comment (ErasedSoul) [reply Backend/app/alchemist/src/Main.hs:70]: Your change is  also okay.. working for we 
`, run-generator all`

Comment (roman-bodavskiy) [reply Backend/app/alchemist/src/Main.hs]: reverted this change, I tested wrong way

**Comment by hkmangla** [comment]
Already merged in main.

**Pull request closed without merging**